applications/app/views/fragments/audioBody.scala.html

```diff
@@ -53,7 +55,7 @@
             ; image <- podcast.image
             ) {
                 <div class="podcast__cover">
-                    <a href="@LinkTo {/@series.id}"><img src="@image" class="podcast__cover-image" alt></a>
+                    <a href="@LinkTo {/@series.id}"><img src="@ImgSrc(image, ImageProfile(width = Some(220)))" class="podcast__cover-image" alt></a>
```

Comment:
I think you want to make this 440 for higher pixel density displays

Reply:
👍